### PR TITLE
New AI names added/previous names alphabetized

### PR DESCRIPTION
**Pull request by rmariani**
I just added some names of AI from Cyberpunk fiction. They are as follows (brackets just their source material, not part of name):

Daedalus (Deus Ex)
Helios (Deus Ex)
Icarus (Deus Ex)
Neuromancer (Neuromancer)
Puppet Master (Ghost in the Shell)
Samaritan (Person of Internets)
Wintermute (Neuromancer)

I also alphabetized some that were at the end in no particular order. I also deleted the the second instance of "Terminus." It was repeated.

### CODE REVIEW

File: config/names/ai.txt [hunk 1]
```diff
@@ -1,157 +1,158 @@
...
+Zord
+Mugsy3000
+Terminus
+Decimus
+Robot Devil
+Optimus
+Megatron
+Soundwave
+Ironhide
+Donald Trump
```
Comment (ghost) [inline config/names/ai.txt:155]: why

**Comment by rmariani** [comment]
Cuz new names and alphabetization

**Comment by Imagebard** [comment]
>Donald Trump

:-1:

**Comment by rmariani** [comment]
Donald Trump was already there

**Comment by clusterfack** [comment]
How delightfully topical

**Comment by Imagebard** [comment]
Are you serious

**Comment by rmariani** [comment]
Yes, it was there before.

**Comment by CptWad** [comment]
I looked it up, @Heredth made it here #555 nearly TWO years ago, before trump was a meme

**Comment by Heredth** [comment]
HAHAHAHAHAH HOLY SHIT

**Comment by Heredth** [comment]
also, looking at my old commits makes me want to fucking vomit

why was i such a shitter jesus christ

**Comment by gbasood** [comment]
Why not:
- Wintermute
- Neuromancer

**Comment by rmariani** [comment]
What's wrong with the other ones?

**Comment by gbasood** [comment]
Nothing, just throwing in suggestions while the PR's open.

**Comment by ghost** [comment]
This isn't even alphabetised

**Comment by rmariani** [comment]
@gbasood I mean they are both there
@DrCelt Yeah I realized

**Comment by gbasood** [comment]
@rmariani are you 100% sure because I tried finding both of those specifically and can't see them
going off the diff, it says 158 adds and 157 removes, or 157 changes and 1 actual additional line - did you forget to commit and push the when you added names or something?

**Comment by clusterfack** [comment]
You got to finish alphabetizing it mate

**Comment by gbasood** [comment]
Yeah, and some of the names you added (wintermute) definitely aren't in there:
https://github.com/rmariani/vgstation13/blob/ai_names/config/names/ai.txt
ctrl+f shows nada

**Comment by rmariani** [comment]
I fucked up, not good at git yet

**Pull request closed without merging**